src/guides/v2.4/graphql/queries/dynamic-blocks.md

```diff
@@ -0,0 +1,156 @@
+---
+group: graphql
+title: dynamicBlocks query
+---
+
+The `dynamicBlocks` query returns a list of dynamic blocks that have been placed in a Dynamic Blocks Rotator inline widget and meet the specified criteria.
+
+{:.bs-callout-info}
+The Banner functionality was removed from Magento 2.4.0 and replaced with dynamic blocks.
+
+When a [Dynamic Blocks Rotator inline widget is created](https://docs.magento.com/user-guide/cms/dynamic-blocks.html), the Magento administrator can select the following options:
```

Comment:
link to https://docs.magento.com/user-guide/cms/dynamic-blocks-rotate.html instead